yarn.lock

```diff
@@ -1547,7 +1549,12 @@
     react-transition-group "^2.4.0"
     truncate "^2.0.1"
 
-"@contentful/forma-36-tokens@^0.9.2", "@contentful/forma-36-tokens@^0.9.4":
+"@contentful/forma-36-tokens@^0.10.0":
+  version "0.10.0"
+  resolved "https://registry.yarnpkg.com/@contentful/forma-36-tokens/-/forma-36-tokens-0.10.0.tgz#7de41cbe31413703aa006cbfc0e3641f37e668e1"
+  integrity sha512-HGl9S1oaA4fb2/9zAAYst1Qjl3N49FzNv0SCuSkSWeXtQag4Q7D8YHe5QzpaeXGvttucXRvQW5iAyBRcYXa/hA==
+
+"@contentful/forma-36-tokens@^0.9.2":
```

Comment:
Are field editors depending on something that uses this or can we deduplicate it away?

Reply:
hmm, let me check

Reply:
everythings seems to be fine, but it keeps appearing here :(

Reply:
I'm getting this with `yarn why`:

```
=> Found "@contentful/field-editor-shared#@contentful/forma-36-tokens@0.9.4"
info This module exists because "_project_#@contentful#field-editor-shared" depends on it.
...
=> Found "@contentful/field-editor-test-utils#@contentful/forma-36-tokens@0.9.4"
info This module exists because "_project_#@contentful#field-editor-test-utils" depends on it.
```

Reply:
ah, fixed that